package.json

```diff
@@ -35,16 +42,19 @@
     "grunt-git": "^1.0.0",
     "grunt-karma": "^2.0.0",
     "grunt-mocha-test": "^0.12.7",
+    "husky": "^0.13.4",
     "injectr": "0.5.1",
     "jasmine-core": "^2.5.1",
     "karma": "^1.3.0",
     "karma-jasmine": "^1.0.2",
     "karma-phantomjs-launcher": "^1.0.2",
     "karma-sauce-launcher": "^1.0.0",
     "karma-sinon": "^1.0.5",
+    "lint-staged": "^3.6.0",
     "load-grunt-tasks": "^3.5.2",
     "mocha": "^3.0.2",
     "phantomjs-prebuilt": "^2.1.12",
+    "prettier-eslint-cli": "^4.0.4",
```

Comment:
@nickbalestra now that we have prettier-eslint-cli and removed eslint directly from the grunt task, shouldn't we remove grunt-eslint as dev dependency, including its own /tasks file?